packages/nimble-components/src/wafer-map/specs/features/grid-dimension.md

```diff
@@ -0,0 +1,28 @@
+# Nimble Wafer Map Grid Dimensions
+
+## Problem Statement
+
+The Wafer Map displays dies on a canvas inside a circular border based on an internal grid which is calculated based on the data input.
+This approach is limiting the performance of the wafer and the grid dimensions can be erroneous if the displayed data is just a subset of a larger data set.
+To mitigate these problems, we will reveal three new API inputs which will allow the users to set the grid dimensions themselves, thus saving processing time and correcting inconsistencies generated by the lack of data context.
```

Comment:
Is this implying that the circular border will be removed when these properties are set? Can you clarify what is the visual impact change as a result of this?

Reply:
no, the border will remain visible. this change will solve an existing issue regarding the way the data points are stretched to cover the whole canvas when they do not have a circular shape. This issue is popping up when the existing wafer values are filtered by value or by the wafer zone, thus warping the display